src/winrtble/utils.rs

```diff
@@ -78,37 +62,8 @@ pub fn to_char_props(_: &GattCharacteristicProperties) -> CharPropFlags {
     CharPropFlags::from_bits_truncate(0 as u8)
 }
 
-#[cfg(test)]
-mod tests {
-    use super::*;
-
-    #[test]
-    fn check_address() {
-        let bluetooth_address = 252566450624623;
-        let addr = to_addr(bluetooth_address);
-        let result = to_address(addr);
-        assert_eq!(bluetooth_address, result);
-    }
-
-    #[test]
-    fn check_uuid_to_guid_conversion() {
```

Comment:
These UUID tests are still useful, please don't delete them.

Reply:
oh right, missed them when the UUID code got moved into its own module

Reply:
I've added them back.